examples/simple/add_header.py

```diff
@@ -1,2 +1,6 @@
-def response(flow):
+from mitmproxy import http
+import typing
+
+
+def response(flow: http.HTTPFlow) -> typing.Any:
```

Comment:
`response()` shouldn't return anything, should it?
At least the [docs](http://docs.mitmproxy.org/en/latest/scripting/events.html#http-events) don't mention anything.
So the correct return type should be `None`.